index.html

```diff
@@ -7,37 +7,192 @@
+              <div class="tonic-title-2">
+                      <h2>Tonic</h2>
+                  <ul class="frame-2">
+                      <li class="canopy-text"><h3>CANOPY</h3></li>
+                      <li class="role-text"><h3>Back End Dev</h3></li>
+                      <li class="year-text"><h3>2015</h3></li>
+                  </ul>
+              </div>
+              <div class="text-paragraph">
+                  <p>
+                      A daily selection of privately personalized reads; 
+                      no accounts or sign-ups required.
+                  </p>
+              </div> 
+              
+              <ul class="primary-buttons">
+                  <li><button class="button1" type="button">html</button></li>
+                  <li><button class="button2" type="button">css</button></li>
+                  <li><button class="button3" type="button">Javascript</button></li>
+              </ul>
+                      <button class="see-project" type="button">See Project</button>
+          </section>
+        </section>
+      
+
+      <section class="tonic-card">
+        <div class="container-image-4">
+        </div>
+          <section class="left-block">                
+              <div class="multi-post">
+                      <h2>Tonic</h2>
+                  <ul class="frame-2">
+                      <li class="canopy-text"><h3>CANOPY</h3></li>
+                      <li class="role-text"><h3>Back End Dev</h3></li>
+                      <li class="year-text"><h3>2015</h3></li>
+                  </ul>
+              </div>
+              <div class="text-paragraph">
+                  <p>
+                      A daily selection of privately personalized reads; 
+                      no accounts or sign-ups required.
+                  </p>
+              </div> 
+              
+              <ul class="primary-buttons">
+                  <li><button class="button1" type="button">html</button></li>
+                  <li><button class="button2" type="button">css</button></li>
+                  <li><button class="button3" type="button">Javascript</button></li>
+              </ul>
+                      <button class="see-project" type="button">See Project</button>
+          </section>
+        </section>
+        
+        <section class="about-me-section">
+            <section>
+                <div class="about-myself">
+                    <h1>About Myself</h1>
+                </div>                
+                <p class="about-me-text">
+                    I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                <div class="link-image">
+                    <p class="connect-link">LET’S CONNECT</p>

```

Comment:
- [ ] Instead of using `p` tag please use the `h2`, `h3`, or `h4` tag to follow the [heading hierarchy](https://www.w3.org/WAI/tutorials/page-structure/headings/). :+1: